src/components/CDPList.js

```diff
@@ -74,25 +157,84 @@ const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
 
+  const [scrollPosition, setScrollPosition] = useState({});
+  const { scrollTop, maxScrollTop } = scrollPosition;
+  const cdpContainerRef = useRef(null);
+
+  const handleOnScroll = () => {
+    const maxScrollTop =
+      cdpContainerRef.current.scrollHeight -
+      cdpContainerRef.current.clientHeight;
+
+    setScrollPosition({
+      ...scrollPosition,
+      scrollTop: cdpContainerRef.current.scrollTop,
+      maxScrollTop
+    });
+  };
+
+  const debounced = debounce(handleOnScroll, 100);
+
+  const scrollAmount = 50;
+  const topPosition = 0;
+  const bottomPosition = 570; //TODO set this to navbarCdps.length * height
+
+  const onDirectionalClick = direction => {
+    let newPosition =
+      direction === 'up' && scrollAmount
+        ? scrollTop - scrollAmount
+        : scrollTop + scrollAmount;
+    if (newPosition < topPosition) newPosition = topPosition;
+    if (newPosition > bottomPosition) newPosition = bottomPosition;
+    setScrollPosition({
+      ...scrollPosition,
+      scrollTop: newPosition,
+      maxScrollTop
+    });
+    cdpContainerRef.current.scrollTop = newPosition;
+  };
+
+  const showDirectionals = navbarCdps.length >= 4;
+  const showTop = showDirectionals && scrollTop > 0;
+  const showBottom = showDirectionals && scrollTop < maxScrollTop;
+
   const { show } = useModal();
 
-  return (
-    <Fragment>
-      {navbarCdps.map((cdp, idx) => {
-        const ratio = ratios[idx] ? round(ratios[idx], 0) : null;
-        const linkPath = `/${Routes.BORROW}/${cdp.id}`;
-        const active = currentPath === linkPath;
-        return (
-          <NavbarItem
-            key={idx}
-            href={linkPath + currentQuery}
-            label={cdp.ilk}
-            owned={account}
-            active={active}
-            ratio={ratio}
-          />
-        );
-      })}
+  return listOpen ? (
+    <Box bg={cdpListFill} height="100%">
+      {showTop && (
+        <DirectionalButton onClick={onDirectionalClick} direction={'up'} />
+      )}
+      <CdpContainer
+        onScroll={debounced}
+        ref={cdpContainerRef}
+        cdpsLength={navbarCdps.length}
+      >
+        <OverviewButton
+          key={navbarCdps.length * 10}
+          href={overviewPath + currentQuery}
+          label={'Overview'}
+          active={active}
+        />
+        {navbarCdps.map((cdp, idx) => {
+          const ratio = ratios[idx] ? round(ratios[idx], 0) : null;
+          const linkPath = `/${Routes.BORROW}/${cdp.id}`;
+          const active = currentPath === linkPath;
+          return (
+            <NavbarItem
+              key={idx}
+              href={linkPath + currentQuery}
+              label={cdp.ilk}
+              owned={account}
+              active={active}
+              ratio={ratio}
+            />
+          );
+        })}
+      </CdpContainer>
+      {showBottom && (
+        <DirectionalButton onClick={onDirectionalClick} direction={'down'} />
```

Comment:
@jparklev one weird UX thing I noticed here, when the buttons disappear, the rest of the component moves up to fill the empty space. This effect reminded me of the website you shared with all the bad UX examples, because the user will accidentally click the wrong button (and open CDP onboarding).

I'm wondering what a better UX approach would be. Maybe fade the buttons in/out slower to give the user more time? Or maybe hide the buttons, but don't remove the element so there is no sudden movement of other elements?